src/test/resources/hw1/AddAndSubtTest.xml

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+<suite name="Addition And Subtract Tests">
```

Comment:
test group suites files should looks like:
```
suite ----
    test------
      classes
```

Use single tag `<test>` with `<classes>` inside